src/view/uniprotkb/components/styles/UniProtTitle.scss

```diff
@@ -4,9 +4,9 @@
     display: inline-flex;
     align-self: center;
     svg {
-      width: 2rem;
-      height: 2rem;
-      top: 0.125em;
+      width: 0.75em;
+      height: 0.75em;
+      margin-right: 0.25rem;
```

Comment:
Is the mixing of `em` and `rem` units intentional here?

Reply:
Yes it 's so the icon size reflects the title font size